picki/app/events/page.js

```diff
@@ -3,16 +3,20 @@
 import { useState } from 'react';
 import ArrowButton from '../components/arrowButton/ArrowButton';
 import TypeFilter from '../components/TypeFilter/TypeFilter';
+import { X } from 'lucide-react';
+import BudgetSlider from '../components/budgetSlider/budgetSlider';
 import './events.css';
 
 
+
 export default function EventPage() {
   // Add the types array to the initial state
   const [eventData, setEventData] = useState({
```

Comment:
Since you have a default event state, maybe move that data into an object, and then pass the object here.

`let defaultEventData = { ... }`

`const [eventData, setEventData] = useState({defaultEventData});